plone/app/locales/locales/it/LC_MESSAGES/plone.po

```diff
@@ -6196,7 +6196,7 @@ msgstr "L'espressione TALES da controllare."
 
 #: CMFPlone/controlpanel/browser/resourceregistry.pt:135
 msgid "The activated Plone-Theme usually provides one CSS bundle and sometimes a javascript bundle."
-msgstr ""
+msgstr "Il tema Plone attivato solitamente include un bundle CSS ed eventualmente un bundle javascript."
```

Comment:
```suggestion
msgstr "Il tema Plone attivato solitamente include un bundle CSS ed eventualmente un bundle JavaScript."
```
Credo sia `JavaScript`, ma e' sbagliato anche l'originale e bisogna vedere quanto e' consistente con il resto delle traduzioni :)

Reply:
Se è quella corretta, meglio cominciare. Nel caso, si cerca "javascript" nelle traduzioni e si cambia in Javascript, no?

Reply:
Come preferite :) 
In ogni caso quello che hai fatto tu e' infinitamente meglio del nulla che c'era prima!
Grazie :+1: